src/components/panes/__tests__/declared-variables-panes.test.js

```diff
@@ -0,0 +1,139 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import SidePane from '../side-pane'
+import { DeclaredVariable } from '../declared-variable'
+import { FrozenVariable } from '../frozen-variable'
+
+import { DeclaredVariablesPaneUnconnected, mapStateToProps } from '../declared-variables-pane'
+import tasks from '../../../actions/task-definitions'
+
+describe('DeclaredVariablesPaneUnconnected React component', () => {
+  let props
+  let mountedPane
+
+  const declaredVariablesPane = () => {
+    if (!mountedPane) {
+      mountedPane = shallow(<DeclaredVariablesPaneUnconnected {...props} />)
+    }
+    return mountedPane
+  }
+
+  beforeEach(() => {
+    props = {
+      userDefinedVarNames: ['iodide', 'a', 'b'],
+      sidePaneMode: 'declared variables',
+      environmentVariables: {
+        x: [
+          'object',
+          'NobwRAhmBcCMAMj4BowCMYCYnwL7PCjngBYBmVDaTATjPNwF0g==',
+        ],
+        y: [
+          'string',
+          'JYewJsYKZA==',
+        ],
+      },
+    }
+    mountedPane = undefined
+  })
+
+  it('always renders one SidePane', () => {
+    expect(declaredVariablesPane().find(SidePane).length).toBe(1)
+  })
+
+  it("sets the HistoryPane's openOnMode prop to be history", () => {
```

Comment:
copy is wrong on this I think - should be `DeclaredVariablesPane` not `HistoryPane`